openstack/ironic/templates/hpa-conductor.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: autoscaling/v1
 kind: HorizontalPodAutoscaler
 metadata:
   name: ironic-conductor
+annotations:
+{{- include "utils.linkerd.pod_and_service_annotation" . | indent 4 }}
 spec:
```

Comment:
I don't think objects of type `HorizontalPodAutoscaler` are in scope for llinkerd. The annotation will probably just do nothing and should imho be removed. I know I had it in the list, but that's because it references a `kind: Deployment` down below. It doesn't seem to have a full spec for that `Deployment`, so it doesn't need the annotation.